web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -238,21 +245,30 @@ internal fun Route.behandlingRoutes(
                         call.svar(resultat)
                     },
                     {
-                        call.audit("Hentet behandling med id ${req.behandlingId}")
+                        call.sikkerlogg("Hentet behandling med id ${req.behandling.id}")
```

Comment:
```suggestion
                        call.sikkerlogg("Lagde brev for behandling med id ${req.behandling.id}")
```

Har ikke noe med denne PRen å gjøre, men må si at jeg her slet litt med å skjønne hva (relatert til brev) denne metoden egentlig gjør. Verken `handleBrevRequest()` eller `søknadsbehandlingService.brev()` hjalp meg nevneverdig.

Reply:
Enig, vi kan kanskje rename den til lagBrevutkast eller noe?